tests/js/spec/views/stream/searchBar.spec.jsx

```diff
@@ -216,10 +216,8 @@ describe('SearchBar', function() {
       };
       let searchBar = mount(<SearchBar {...props} />).instance();
       searchBar.updateAutoCompleteItems();
-      expect(searchBar.state.searchTerm).to.eql('unresolved');
-      expect(searchBar.state.searchItems.length).to.eql(1);
-      expect(searchBar.state.searchItems[0].desc).to.eql('unresolved');
-      expect(searchBar.state.searchItems[0].value).to.eql('unresolved');
+      expect(searchBar.state.searchTerm).to.eql('fu');
+      expect(searchBar.state.searchItems.length).to.eql(0);
```

Comment:
The original assertions on this test were ... completely wrong, and I'm not sure what I was doing when I wrote them. The new assertions matches the behavior from the live app today.